en/streaming-integrator/docs/connectors/downloading-and-Installing-Siddhi-Extensions.md

```diff
@@ -55,14 +54,63 @@ To uninstall a Siddhi extension, delete the relevant extension JAR in the `<SI_H
 
 To manage Siddhi extensions via the command line, see the following topics.
 
-### Finding Siddhi extensions to install
+### Identifying the Siddhi extensions to install/uninstall
 
-The following are some actions that you are required to perform in order to determine which Siddhi extensions you need to install.
+The following are some actions that you are required to perform in order to determine which Siddhi extensions you need to install. Navigate to the `<SI_HOME>/wso2/tools/extension-installer` in order to issue these commands.
 
-|**Required Action**                                                        |**CLI Command**        |
-|---------------------------------------------------------------------------|-----------------------|
-|
+- **Viewing the list of extensions that are currently installed**
 
+    You can view the complete list of Siddhi extensions that are currently installed in your Streaming Integrator setup. All the extensions listed are completely installed with the dependencies.
+    
+    To perform this action, issue the appropriate command out of the following based on your operating system:
+    
+    - **For Windows**     : `extension-installer.bat list`
+    - **For Linux/MacOS** : `./extension-installer.sh list`
+    
+- **Viewing the installation status of all the supported Siddhi extensions**
 
+    You can view the complete list of Siddhi extensions supported for WSO2 Streaming Integrator together with the current installation status for each extension.
+    
+    The installation status can be one of the following:
+    
+    |**Installation Status**|**Description**                                                    |
+    |-----------------------|-------------------------------------------------------------------|
+    |**Installed**          |This indicates that the extension is completely installed. The installation includes the JAR of the extension itself as well as all its dependencies (if any).|
+    |**Not-Installed**      |This indicates that the extension has not been installed. The JAR of the extension itself has not been installed. Dependencies (if any) may be already installed due to shared dependencies.|
+    |**Partially-Installed**|This indicates that the JAR of the extension itself has been installed, but one or more dependencies of the extension still need to be installed.<br/><br/> If an extension has this status, you can view more information about the dependencies to be installed by checking the installation status of that specific extension individually.|
+    |**Restart-Required**   |This indicates that you need to restart Streaming Integrator Tooling in order to complete the installation/un-installation of the extension.|
```

Comment:
Let's remove the 'Restart-Required' Installation status from here, it's only applicable for the UI in tooling runtime.
In this tool we just inform the user to restart, after an installation/un-installation.